R/simulation-results.R

```diff
@@ -3,19 +3,55 @@
 #' @docType class
 #' @description  Results of a simulation run (either individual or population simulation)
 #'
-#' @field count the number of individual results (\code{Count==1} generally means that we are dealing with an individual simulation results)
-#' @field simulation Reference to the \code{Simulation} used to calculate or import the results (Read-Only)
+#' @field count the number of individual results (\code{Count==1} generally means that we are dealing with an individual simulation results).
+#' @field simulation Reference to the \code{Simulation} used to calculate or import the results (Read-Only).
+#' @field timeValues Vector of simulated time output values
+#' @field allQuantityPaths List of all paths for which results are defined.
+#' @field allIndividualId List of Ids of all individuals that have been simulated
 #' @format NULL
 SimulationResults <- R6::R6Class("SimulationResults",
   inherit = DotNetWrapper,
   private = list(
-    .simulation = NULL
+    .simulation = NULL,
+    getResultsForIndividual = function(individualId) {

```

Comment:
I moved these to `private` as they return .NET objects and we don't want to expose them..